packages/wallets/src/public/images/arrow-bold.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><g fill="var(--fill-color1)" fill-rule="evenodd"><path fill="none" d="M8 1L1 8l7 7M1 8h14"/><path fill-rule="nonzero" d="M3.414 7H15a1 1 0 010 2H3.414l5.293 5.293a1 1 0 01-1.414 1.414l-7-7a1 1 0 010-1.414l7-7a1 1 0 111.414 1.414L3.414 7z"/></g></svg>
```

Comment:
We have this icon already right? `ic-back-arrow.svg`

Reply:
Yes, will change 😅 .